ansible/config_sonic_basedon_testbed.yml

```diff
@@ -543,6 +543,13 @@
       - name: execute cli "config load_minigraph --override_config -y" to apply new minigraph
         become: true
         shell: config load_minigraph --override_config -y
+        register: load_minigraph_rc
+        ignore_errors: yes
```

Comment:
this will not just ignore the error that you want to ignore (`override_config` not supported). This will also ignore all other errors.

If any other error happens, the code will still continue to do `config load_minigraph -y`.  That's not what you expect, right?